src/sentry/api/serializers/models/activity.py

```diff
@@ -19,7 +21,16 @@ def get_attrs(self, item_list, user):
         }
         if commit_ids:
             commit_list = list(Commit.objects.filter(id__in=commit_ids))
-            commits_by_id = {c.id: d for c, d in zip(commit_list, serialize(commit_list, user))}
+            commits_by_id = {
+                c.id: d
+                for c, d in zip(
+                    commit_list,
+                    # serialize with just commits and no associated releases
+                    # serialize(commit_list, user, serializer=CommitSerializer()),
+                    # serialize with commit with any associated releases
+                    serialize(commit_list, user, serializer=CommitWithReleaseSerializer()),
```

Comment:
@wedamija there's probably a better way than to init the serializer every time this method is called.

Reply:
Nah that's fine, it's only instantiated once per call the the endpoint so shouldn't be a problem.